appveyor.yml

```diff
@@ -18,45 +49,37 @@ install:
   - ps: '[IO.Compression.ZipFile]::ExtractToDirectory("C:\winium.zip", "C:\winium")'
 
 clone_script:
-- ps: git clone -q $("--branch=" + $Env:APPVEYOR_REPO_BRANCH) $("https://github.com/" + $Env:APPVEYOR_REPO_NAME + ".git") $Env:APPVEYOR_BUILD_FOLDER
-- ps: if (!$Env:APPVEYOR_PULL_REQUEST_NUMBER) {$("git checkout -qf " + $Env:APPVEYOR_REPO_COMMIT)}
-- ps: if ($Env:APPVEYOR_PULL_REQUEST_NUMBER) {git fetch -q origin +refs/pull/$($Env:APPVEYOR_PULL_REQUEST_NUMBER)/merge; git checkout -qf FETCH_HEAD}
+- ps: git clone -q $("--branch=" + $env:APPVEYOR_REPO_BRANCH) $("https://github.com/" + $env:APPVEYOR_REPO_NAME + ".git") $env:APPVEYOR_BUILD_FOLDER
+- ps: if (!$env:APPVEYOR_PULL_REQUEST_NUMBER) {$("git checkout -qf " + $env:APPVEYOR_REPO_COMMIT)}
+- ps: if ($env:APPVEYOR_PULL_REQUEST_NUMBER) {git fetch -q origin +refs/pull/$($env:APPVEYOR_PULL_REQUEST_NUMBER)/merge; git checkout -qf FETCH_HEAD}
 - ps: git submodule update -q --init --recursive
 
 before_build:
-- ps: $env:playgroundNet46_dir=$Env:APPVEYOR_BUILD_FOLDER + "\ReactWindows\Playground.Net46"
-- ps: $env:bundle_dir=$Env:APPVEYOR_BUILD_FOLDER + "\ReactWindows\Playground.Net46\ReactAssets"
+- ps: $env:playgroundNet46_dir=$env:APPVEYOR_BUILD_FOLDER + "\ReactWindows\Playground.Net46"
+- ps: $env:bundle_dir=$env:APPVEYOR_BUILD_FOLDER + "\ReactWindows\Playground.Net46\ReactAssets"
 - ps: nuget restore ReactWindows\ReactNative.sln
 - npm i -g react-native-cli
 
 build_script:
 - ps: mkdir $env:bundle_dir
 - ps: react-native bundle --platform windows --entry-file $($env:playgroundNet46_dir + "\index.windows.js") --bundle-output $($env:bundle_dir + "\index.windows.bundle") --assets-dest $env:bundle_dir --dev false; echo "Suppressing error"
-- cmd: >-
-    set DEVENV="%VS140COMNTOOLS%\..\IDE\devenv"
-
-    %DEVENV% /build "Debug|x86" ReactWindows\ReactNative.sln
-
-    %DEVENV% /build "ReleaseBundle|x64" ReactWindows\ReactNative.sln
-
-    %DEVENV% /build "Debug|ARM" ReactWindows\ReactNative.sln
+- ps: msbuild /p:Configuration=$env:CONFIGURATION /p:Platform=$env:PLATFORM /p:PlatformToolset=$env:PLATFORM_TOOLSET /nologo /logger:"C:\Program Files\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll" "ReactWindows\ReactNative.sln"
 
 # Start Winium in the background, give it a moment to start
 before_test:
   - ps: $winium = Start-Process -PassThru C:\winium\Winium.Desktop.Driver.exe
   - ps: Start-Sleep -s 5
 
-test:
-  assemblies:
-  - ReactWindows\ReactNative.Net46.Tests\bin\x64\**\*.Tests.dll
-
 test_script:
-#  - npm run flow-check
-  - npm test
+  # Run Nunit 3.x test engine and output to AppVeyor's UI.
+  - ps: nunit3-console "ReactWindows\ReactNative.Net46.Tests\bin\$env:PLATFORM\$env:CONFIGURATION\ReactNative.Net46.Tests.dll" --result=myresults.xml
```

Comment:
this will not work for the ARM configuration -- the tests will fail to run due to platform mismatch. that's why we had the single path to the x64 binaries before.